language-extensions/python/README.md

```diff
@@ -25,6 +25,8 @@ Note that the Python Extension released in the current repository works with Pyt
 
 1. Modify [**CMakeLists.txt**](src/CMakeLists.txt). Change each `find_library` call to point to your custom python and boost libraries. 
 
+1. Modify [**PythonExtension.cpp**](src/PythonExtension.cpp). Change the value of `x_PythonSoFile` on line 37 to the name of your python so library file.
+

```

Comment:
You should revert the change in this file. This section should be under linux since it is x_PythonSoFile